### ISSUE TITLE
suggest to add `-n EDTA` for `conda install`

### ISSUE THREAD

**Issue by zhangrengang** [oushujun/EDTA#26]
My conda version is `Miniconda2 4.4.10`, whose `base` use `python2`. `conda install python=3.6` damaged the env of `base`, following import error for conda:
```
$ conda -h
Traceback (most recent call last):
  File "~/conda/bin/conda", line 7, in <module>
    from conda.cli import main
ImportError: No module named conda.cli
```
So it is better to not install `python=3.6` in conda's `base`.

**Comment by oushujun** [comment]
Hi Ren-Gang,

The installation first creates the "EDTA" env (`conda create -n EDTA`) then activate it. The following `conda install` will be installed in the EDTA env. Do you mean it was not automatically installed in the EDTA env under `Miniconda2 4.4.10`?

Best,
Shujun

**Comment by zhangrengang** [comment]
@oushujun, yes, it is automatically installed in `base`. It might be beacuse `conda activate EDTA` failed but I ignored it. Thank you.

**Comment by oushujun** [comment]
I see. I have add `-n EDTA` to each `conda install` line. This will be reflected in the next update. Thanks for your suggestion and sorry for the inconvenience. 

Shujun

**Comment by zhangrengang** [comment]
Thank you.
